packages/front-end/components/Features/FeatureModal/index.tsx

```diff
@@ -357,6 +366,22 @@ export default function FeatureModal({
           released to users.
         </div>
       )}
+      {projectId === selectedProject && (
+        <div className="alert alert-warning">
+          You are creating a feature under the demo datasource project.
+        </div>
```

Comment:
This is correct, but it's a little confusing to read. We also have a duplicate warning at the top of this modal. I'd remove the duplicate warning at the top of this modal, and then update this to be:

```ts
// destructure and rename projectId so it's a bit easier to understand
const { projectId: demoProjectId } = useDemoDataSourceProject();
...
  {selectedProject === demoProjectId && (
       <div className="alert alert-warning">
        // Updates the warning message to add some context around why a user might not want to create a feature in the demo project
          You are creating a feature under the demo datasource project. This
          feature will be deleted when the demo datasource project is deleted.
       </div>
   )}
```